core/controllers/base_test.py

```diff
@@ -87,6 +87,18 @@ def test_requests_for_invalid_paths(self):
         response = self.testapp.put('/gallery/extra', {}, expect_errors=True)
         self.assertEqual(response.status_int, 404)
 
+    def test_redirect_in_both_logged_in_and_logged_out_states(self):
+        "Test for a redirect in both logged in and logged out states on '/'."
+
+        # Logged out state
+        response = self.testapp.get('/')
+        self.assertEqual(response.status_int, 302)
```

Comment:
@seanlip
* For the 1st case (logged out user), checking for `self.testapp.get('/').headers['location']`, contains `gallery`, as it should.

* But for the 2nd case (logged in user), after the `self.login('user@example.com')`, when I check for `self.testapp.get('/').headers['location']`, then it's value is `https://www.google.com/accounts/Logout?continue=http%3A//localhost/` 

I'm not getting what is happening here, since after the login, shouldn't the url location be the one for `my_explorations` ?